h/cli/commands/search.py

```diff
@@ -11,27 +8,6 @@ def search():
     """Manage search index."""
 
 
-@search.command()
-@click.pass_context
-def reindex(ctx):
```

Comment:
I don't think we need this or that it would work now.

Reply:
:+1: